web/mass_class/rtmp_bridge.py

```
@@ -0,0 +1,178 @@
+            await database_sync_to_async(self._update_stream_status)(
+                session_id, "active", f"https://www.youtube.com/watch?v={youtube_key}"
+            )
+
+            # Notify teacher that streaming has started
+            await self.channel_layer.send(
+                teacher_channel,
+                {"type": "stream_started", "youtube_url": f"https://www.youtube.com/watch?v={youtube_key}"},
+            )
+
+            # Monitor process in background
+            asyncio.create_task(self._monitor_process(session_id, teacher_channel))
```

Ruff (0.8.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)